ShopLens/ShopLensForms/IO/IReader.cs

```diff
@@ -5,10 +5,14 @@ namespace ShopLensApp.IO
     interface IReader
     {
         /// <summary>
-        /// Deserialize string of JSON format to the list of 'Item' objects.
+        /// Deserialize string to the list of 'Item' objects.
```

Comment:
Just a note - you can usee <see cref="ShoppingItem"/>. That will be a direct reference to code